Userland/Utilities/seq.cpp

```diff
@@ -98,7 +101,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             else if ((dot - buf) + 1 + number_of_decimals < (int)sizeof(buf))
                 dot[1 + number_of_decimals] = '\0';
         }
-        outln("{}", buf);
+        out("{}{}", buf, separator);
```

Comment:
In other implementations of `seq`, the separator only appears between the numbers, not after the last one. It would be better if we were consistent with those.

Reply:
I modeled this on the MacOS implementation, where `seq -s a 1 2` returns `1a2a`. That said, I'm happy to change this if we prefer the other approach.

Reply:
Ah ok. No, you can keep it as it is then.